Cargo.toml

```diff
@@ -8,7 +8,14 @@ license = "MIT OR Apache-2.0"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-warp = {version = "0.2", features = ["tls"] }
+async-graphql = "1.13.2"
+async-graphql-warp = "1.13.2"
+diesel = { version = "1.4.4", features = ["sqlite", "r2d2"] }
+dotenv = "0.9.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0.53"
 tokio = { version = "0.2.21", features = ["macros"] }
-serde = {version = "1.0", features = ["derive"]}
-serde_json = "1.0.53"
\ No newline at end of file
+warp = { version = "0.2", features = ["tls"] }
+
+[target.'cfg(windows)'.dependencies]
+libsqlite3-sys =  { version = "0.9.1", features = ["bundled"] }
```

Comment:
I had to use a bundled version when compiling for windows. Couldn't make gh actions install the library.

Reply:
Well, since I develop in windows and we have all platforms tests I wanted to make it work 😅